docs/prod-maintain-secure-upgrade.md

```diff
@@ -416,9 +416,8 @@ the `knight` management command:
 
 If you need to manage the IRC, Jabber, or Zephyr mirrors, you will
 need to create API super users.  To do this, use `./manage.py knight`
-with the `--permission=api_super_user` argument.  See
-`api/integrations/irc-mirror.py` and
-`api/integrations/jabber_mirror.py` for further detail on these.
+with the `--permission=api_super_user` argument.  See the respective
+integration scripts for these mirrors for further detail on these.
```

Comment:
Yeah, this section is never going to be particularly clear without actual docs somewhere for these mirrors, but that'd help a bit.